app/controllers/validation_rules_controller.rb

```
@@ -35,68 +35,69 @@ def new
       @forms = Form.accessible_by(current_ability).where(mission: @mission)
       @questions = @validation_rule.form&.questions || []
-      render :edit
+      render(:edit)
     end
   end
 
   def destroy
     authorize!(:destroy, @validation_rule)
-    
+
     @validation_rule.destroy
```

RuboCop (1.82.0)

[convention] Use `destroy!` instead of `destroy` if the return value is not checked.

(Rails/SaveBang)